src/scripts/main.js

```diff
@@ -1,3 +1,315 @@
+
+form.className = 'new-employee-form';
+
+const inputName = document.createElement('input');
+const labelName = document.createElement('label');
+
+inputName.type = 'text';
+inputName.setAttribute('data-qa', 'name');
+inputName.setAttribute('required', '');
+labelName.textContent = 'Name:';
+labelName.appendChild(inputName);
+form.append(labelName);
+
+const inputPosition = document.createElement('input');
+const labelPosition = document.createElement('label');
+
+inputPosition.type = 'text';
+inputPosition.setAttribute('data-qa', 'position');
+inputPosition.setAttribute('required', '');
+labelPosition.textContent = 'Position:';
+labelPosition.appendChild(inputPosition);
+form.append(labelPosition);
+
+const inputOffice = document.createElement('select');
+const labelOffice = document.createElement('label');
+
+inputOffice.setAttribute('data-qa', 'office');
+inputOffice.setAttribute('required', '');
+labelOffice.textContent = 'Office:';
+
+const select = [
+  { value: 'Tokyo', text: 'Tokyo' },
+  { value: 'Singapore', text: 'Singapore' },
+  { value: 'London', text: 'London' },
+  { value: 'New York', text: 'New York' },
+  { value: 'Edinburgh', text: 'Edinburgh' },
+  { value: 'San Francisco', text: 'San Francisco' },
+];
+
+select.forEach((selectData) => {
+  const option = document.createElement('option');
+
+  option.value = selectData.value;
+  option.textContent = selectData.text;
+  inputOffice.append(option);
+});
+
+labelOffice.appendChild(inputOffice);
+form.append(labelOffice);
+
+const inputAge = document.createElement('input');
+const labelAge = document.createElement('label');
+
+inputAge.type = 'number';
+inputAge.setAttribute('data-qa', 'age');
+inputAge.setAttribute('required', '');
+labelAge.textContent = 'Age:';
+labelAge.appendChild(inputAge);
+form.append(labelAge);
+
+const inputSalary = document.createElement('input');
+const labelSalary = document.createElement('label');
+
+inputSalary.type = 'number';
+inputSalary.setAttribute('data-qa', 'salary');
+inputSalary.setAttribute('required', '');
+labelSalary.textContent = 'Salary:';
+labelSalary.appendChild(inputSalary);
+form.append(labelSalary);
+
+const button = document.createElement('button');
+
+button.type = 'submit';
+button.textContent = 'Save to table';
+form.append(button);
+
+document.body.append(form);
+
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const existingNotification = document.querySelector('.notification');
+
+  if (existingNotification) {
+    existingNotification.remove();
+  }
+
+  const notification = document.createElement('div');
+
+  notification.className = 'notification';
+
+  notification.setAttribute('data-qa', 'notification');
+
+  let isValid = true;
+
+  if (
+    inputName.value.trim().length < 4 ||
+    !/^[a-zA-Z\s]+$/.test(inputName.value)
+  ) {
+    isValid = false;
+    notification.classList.add('error');
+
+    notification.textContent =
+      'Error! Name value should have more than 4 letters!';
```

Comment:
The validation message states that the name should have more than 4 letters, but the condition checks for less than 4. Ensure the condition aligns with the message, i.e., it should be `inputName.value.trim().length <= 3`.